x/btcstaking/keeper/inclusion_proof.go

```diff
@@ -10,23 +10,29 @@ import (
 	"github.com/babylonlabs-io/babylon/x/btcstaking/types"
 )
 
+type delegationTimeRangeInfo struct {
+	startHeight uint32
+	endHeight   uint32
+}
+
 // VerifyInclusionProofAndGetHeight verifies the inclusion proof of the given staking tx
-// and returns the inclusion height
+// and returns the start height and end height
 func (k Keeper) VerifyInclusionProofAndGetHeight(
 	ctx sdk.Context,
 	stakingTx *btcutil.Tx,
+	confirmationDepth uint32,
 	stakingTime uint32,
+	minUnbondingTime uint32,
 	inclusionProof *types.ParsedProofOfInclusion,
-) (uint32, error) {
-	btccParams := k.btccKeeper.GetParams(ctx)
+) (*delegationTimeRangeInfo, error) {
```

Comment:
How about  

```suggestion
) (uint32, uint32, error) {
```

as this object is [unpacked](https://github.com/babylonlabs-io/babylon/pull/218/files#diff-0490899e13cee0223ce9b01147d9ad2753112c4e8118443e1b54684738fc4969R201-R202) to two uint32 later anyway?

Reply:
So this may be a personal preference, but:
- I really dislike returning more that 2 values from one function, it makes is hard to read and call.
- `(uint32, uint32, error) ` does not tell you much what those `uint32` represent while in returned struct those numbers have proper names `startHeight uint32` which makes life easier for the caller